src/styles/folders-modal.scss

```diff
@@ -0,0 +1,112 @@
+@at-root .snippetor-folders-modal#{&} .snippetor-content {
+  .contains-task-list {
```

Comment:
You should not be overlaying task-list, I don't think. The same comments as above, I think this should all be emulating .nav classes.. (or making your own class)